_studio/mfx_lib/Android.mk

```diff
@@ -0,0 +1,208 @@
+LOCAL_PATH:= $(MFX_HOME)/_studio
+
+# =============================================================================
+
+MFX_LOCAL_DECODERS := h265 h264 mpeg2 vc1 mjpeg vp8
+MFX_LOCAL_ENCODERS := h264 mjpeg
+
+# Setting subdirectories to march thru
+MFX_LOCAL_DIRS := \
+    scheduler \
+    fei
+
+MFX_LOCAL_DIRS_IMPL := \
+    $(addprefix decode/, $(MFX_LOCAL_DECODERS)) \
+    vpp
+
+MFX_LOCAL_DIRS_HW := \
+    $(addprefix encode_hw/, $(MFX_LOCAL_ENCODERS)) \
+    genx/h264_encode \
+    cmrt_cross_platform
+
+MFX_LOCAL_SRC_FILES := \
+    $(patsubst $(LOCAL_PATH)/%, %, $(foreach dir, $(MFX_LOCAL_DIRS), $(wildcard $(LOCAL_PATH)/mfx_lib/$(dir)/src/*.cpp)))
+
+MFX_LOCAL_SRC_FILES_IMPL := \
+    $(patsubst $(LOCAL_PATH)/%, %, $(foreach dir, $(MFX_LOCAL_DIRS_IMPL), $(wildcard $(LOCAL_PATH)/mfx_lib/$(dir)/src/*.cpp)))
+
+MFX_LOCAL_SRC_FILES_HW := \
+    $(MFX_LOCAL_SRC_FILES_IMPL) \
+    $(patsubst $(LOCAL_PATH)/%, %, $(foreach dir, $(MFX_LOCAL_DIRS_HW), $(wildcard $(LOCAL_PATH)/mfx_lib/$(dir)/src/*.cpp)))
+
+MFX_LOCAL_C_INCLUDES := \
+    $(foreach dir, $(MFX_LOCAL_DIRS), $(wildcard $(LOCAL_PATH)/mfx_lib/$(dir)/include))
+
+MFX_LOCAL_C_INCLUDES_IMPL := \
+    $(MFX_LOCAL_C_INCLUDES) \
+    $(foreach dir, $(MFX_LOCAL_DIRS_IMPL), $(wildcard $(LOCAL_PATH)/mfx_lib/$(dir)/include))
+
+MFX_LOCAL_C_INCLUDES_HW := \
+    $(MFX_LOCAL_C_INCLUDES_IMPL) \
+    $(foreach dir, $(MFX_LOCAL_DIRS_HW), $(wildcard $(LOCAL_PATH)/mfx_lib/$(dir)/include)) \
+    $(MFX_HOME)/_studio/mfx_lib/genx/field_copy/include \
+    $(MFX_HOME)/_studio/mfx_lib/genx/copy_kernels/include
+
+MFX_LOCAL_STATIC_LIBRARIES_HW := \
+    libmfx_lib_merged_hw \
+    libumc_codecs_merged_hw \
+    libumc_codecs_merged \
+    libumc_io_merged_hw \
+    libumc_core_merged_hw \
+    libmfx_trace_hw \
+    libsafec
+
+MFX_LOCAL_LDFLAGS_HW += \
+    $(MFX_LDFLAGS) \
+    -Wl,--version-script=$(LOCAL_PATH)/mfx_lib/libmfx.map \
+    -Wl,--no-warn-shared-textrel
```

Comment:
Here is one more. I think we need a grep results  to eliminate all such warning flags in Android.mk files